gnome-initial-setup/gis-driver.c

```diff
@@ -312,6 +316,7 @@ gis_driver_startup (GApplication *app)
 
   priv->main_window = gis_window_new (driver);
   priv->assistant = gis_window_get_assistant (GIS_WINDOW (priv->main_window));
+  priv->music_player = gis_music_player_new ("file://" PKGDATADIR "/fbe.ogg");
```

Comment:
It is not, pending approval from Dalio. I can put in the temporary asset for now.